market_maker_keeper/zrx_market_maker_keeper.py

```diff
@@ -119,6 +119,9 @@ def __init__(self, args: list, **kwargs):
         parser.add_argument('--cancel-on-shutdown', dest='cancel_on_shutdown', action='store_true',
                             help="Whether should cancel all open orders on keeper shutdown")
 
+        parser.add_argument("--remember-own-orders", dest='debug', action='store_true',
```

Comment:
I think `dest='debug'` will actually make it not work as expected

Reply:
was looking why I got ```AttributeError: 'Namespace' object has no attribute 'remember_own_orders'
``` and now I know, thks for spotting that !